configs/platforms/sles-11-x86_64.rb

```diff
@@ -0,0 +1,22 @@
+platform "sles-11-x86_64" do |plat|
+  plat.servicedir "/etc/init.d"
+  plat.defaultdir "/etc/sysconfig"
+  plat.servicetype "sysv"
+
+  plat.add_build_repository "http://osmirror.delivery.puppetlabs.net/sles-11-deps-x86_64/sles-11-deps-x86_64.repo"
+  plat.add_build_repository "http://pl-build-tools.delivery.puppetlabs.net/yum/sles/11/x86_64/pl-build-tools-sles-11-x86_64.repo"
+  packages = %w[
+    aaa_base
+    libbz2-devel
+    pkgconfig
+    pl-autotools
+    pl-cmake
+    pl-gcc8
```

Comment:
There's some duplication here and `runtime` ... Seems like these build dependencies should be in one place or the other but not both.

Reply:
Ohh yess!

Reply:
I think it's fine, zypper will see that it's already installed